gateway-server/src/test/java/org/apache/knox/gateway/session/control/InMemoryConcurrentSessionVerifierTest.java

```diff
@@ -316,25 +296,27 @@ public void testBackgroundThreadRemoveExpiredTokens() throws ServiceLifecycleExc
     verifier.verifySessionForUser("admin", adminToken2);
     JWT expiringAdminToken = tokenAuthority.issueToken(expiringJwtAttributesForAdmin);
     verifier.verifySessionForUser("admin", expiringAdminToken);
-    Assert.assertEquals(3, verifier.countValidTokensForUser("admin"));
-    Thread.sleep(1100);
-    Assert.assertEquals(2, verifier.countValidTokensForUser("admin"));
+    Assert.assertEquals(3, verifier.getTokenCountForUser("admin").intValue());
+    Thread.sleep(1050);
```

Comment:
If you are invoking `verifier.removeExpiredTokens();` anyway, why do we need the Thread.sleep here?

Reply:
It is for waiting until the token expires, but i realised i could just issue already expired tokens and skip the waiting. That would probably check the same functionality. I will change it that way and remove the sleep (at the other places as well)